c2rust-bitfields-derive/src/lib.rs

```diff
@@ -228,6 +232,17 @@ fn bitfield_struct_impl(struct_item: ItemStruct) -> Result<TokenStream, Error> {
     let field_bit_info = field_bit_info?;
     let field_bit_info_setters = &field_bit_info;
     let field_bit_info_getters = &field_bit_info;
+    let field_endians: Vec<_> = bitfields
+        .iter()
+        .map(|field| {
+            field
+                .endian
+                .as_ref()
+                .map_or(false, |endian| endian == "big")
```

Comment:
Should it only check for `big` or should it have to be `big`, `little`, or nothing?

Reply:
endian could be an enum, and if it had a default, this could be `.unwrap_or_default()`. (I lack the context here to understand whether a default endianness makes sense at all; gut says no).